Create IGSR superpopulations and reclassify reference populations

aleixpuigb opened the pull request.

aleixpuigb committed:
Create IGSR superpopulations in the reference populations table

aleixpuigb committed:
Create all superpopulations

jamesamcl commented:
@aleixpuigb can we tag the new terms as AI assisted?

aleixpuigb committed:
Update classification for reference populations

aleixpuigb committed:
Correct project for Masai in Kenya

aleixpuigb committed:
Add superpopulation code to 1KG datasets, and long name as synonym

aleixpuigb committed:
Fix wrong AP for SAS reference population

daniwelter commented:
@aleixpuigb How quickly do you need this turned around? A couple of these reclassifications aren't entirely straightforward, e.g. the Gambian populations didn't get classified at all while two of the SGDP populations actually contain samples from other collections, so need closer inspections to ensure correct classification. There is also the issue of the SGPD "West Eurasia" label, which raised concerns in the past, so I want to do a bit more reading before deciding on its inclusion. 

If it's very urgent, we can probably aim for a release before the end of the month but if it can wait a bit longer, there's a couple of other changes in the queue that I'd like to include if possible.

aleixpuigb committed:
Change to the abbreviation 1KGP

aleixpuigb commented:
Hi @daniwelter, just checking in to see if you had any thoughts on my earlier comment about project-scoped superpopulations.

aleixpuigb committed:
Update IDs

aleixpuigb committed:
Update IDs in reference_populations.tsv

Renumbered HANCESTRO reference superpopulation IDs in reference_populations.tsv.

daniwelter committed:
Further updates to superpopulations. Fixes #67. Typo corrections; fixes #68. NTRs addressed; fixes #69 #70.

daniwelter committed:
Further corrections to the reference population hierarchy in line with scope of #67

daniwelter committed:
Release v2025-10-14 fully prepared

daniwelter committed:
Final minor fix before release

daniwelter reviewed (approved):
Changes reviewed and approved
